exporter/sematextexporter/README.md

```diff
@@ -0,0 +1,54 @@
+# Sematext Exporter
+<!-- status autogenerated section -->
+<!-- end autogenerated section -->
+
+This exporter supports sending metrics to [Sematext Cloud](https://sematext.com/) in Influx line protocol format
+
+## Configuration
+
+The following configuration options are supported:
+
+* `endpoint` (required) HTTP/S destination for metric receivers. It is dependent on the region:
+    - US: `spm-receiver.sematext.com`
+    - EU: `spm-receiver.eu.sematext.com`
+* `timeout` (default = 5s) Timeout for requests
+* `AppToken` App token is the token of Sematext Monitoring App to which you want to send the metrics.
+* `Region`  Region specifies the Sematext region the user is operating in; must be one of:
+  * `US`
+  * `EU`
+* `payload_max_lines` (default = 10_000) Maximum number of lines allowed per HTTP POST request
+* `payload_max_bytes` (default = 10_000_000) Maximum number of bytes allowed per HTTP POST request
```

Comment:
@Eromosele-SM I'd check these defaults internally with Sematext first.

Reply:
Is this it?
https://sematext.com/docs/monitoring/custom-metrics/#:~:text=We%20recommend%20sending%20batches%20of%20between%20100%20and%20500%20lines%20of%20metrics.

Reply:
That's the suggest for custom metrics. I'd ask STA devs what STA uses.